### DISCUSSION TITLE
Nativescript support for Angular Nested Tree Structure

### DISCUSSION THREAD

**Discussion by veeravetrivel** [NativeScript/NativeScript#9640] [Q&A]
I need to perform UI nested tree like structure with multi level parent to children. 

Example: https://angular2-tree.readme.io/

Is there any component or NPM to support tree component? if yes let me know to handle the component. 
@alg  @ligaz @kant @madmas @hdeshev

**Comment by Ross-Olivares** [comment]
Hi @veeravetrivel 
Thanks for reporting.
I am discussing this with the team, will update as soon as I have some information.
